registration/locale/de/LC_MESSAGES/django.po

```diff
@@ -59,70 +82,82 @@ msgstr ""
 msgid "email address"
 msgstr "E-Mail-Adresse"
 
-#: registration/models.py:76
+#: registration/models.py:73
+msgid "Media Authority"
+msgstr "Bürgermedium"
+
+#: registration/models.py:101
 msgid "first name"
 msgstr "Vorname"
 
-#: registration/models.py:78
+#: registration/models.py:103
 msgid "last name"
 msgstr "Nachname"
 
-#: registration/models.py:83
+#: registration/models.py:108
 msgid "Not Given"
 msgstr "Nicht angegeben"
 
-#: registration/models.py:84
+#: registration/models.py:109
 msgid "Male"
 msgstr "Männlich"
 
-#: registration/models.py:85
+#: registration/models.py:110
 msgid "Female"
 msgstr "Weiblich"
 
-#: registration/models.py:86
+#: registration/models.py:111
 msgid "Diverse"
 msgstr "Divers"
 
-#: registration/models.py:89
+#: registration/models.py:114
 msgid "gender"
 msgstr "Geschlecht"
 
-#: registration/models.py:98
+#: registration/models.py:123
 msgid "phone number"
 msgstr "Telefonnummer"
 
-#: registration/models.py:101
+#: registration/models.py:126
 msgid "mobile number"
 msgstr "Handynummer"
 
-#: registration/models.py:105
+#: registration/models.py:130
 msgid "birthday"
 msgstr "Geburtstag"
 
-#: registration/models.py:108
+#: registration/models.py:133
 msgid "street"
 msgstr "Straße"
 
-#: registration/models.py:110
+#: registration/models.py:135
 msgid "house number"
 msgstr "Hausnummer"
 
-#: registration/models.py:112
+#: registration/models.py:137
 msgid "zipcode"
 msgstr "Postleitzahl"
 
-#: registration/models.py:114
+#: registration/models.py:139
 msgid "city"
 msgstr "Stadt"
 
-#: registration/models.py:118
+#: registration/models.py:143
 msgid "verified"
 msgstr "verifiziert"
 
-#: registration/models.py:120
+#: registration/models.py:145
 msgid "The profile data was verified by showing the ID to an employee."
 msgstr "Das Profil wurde unter Vorlage eines Personalausweises verifiziert."
 
+#: registration/print.py:81
+msgid "registration_form.pdf"
+msgstr "Nutzer*innenantrag.pdf"
```

Comment:
If this really is the file name of the downloaded file we might want to have the underscore here for compatibility reasons with file system tools.
```suggestion
msgid "registration_form.pdf"
msgstr "Nutzer_innenantrag.pdf"
```